packages/components/modal/src/Modal.tsx

```diff
@@ -15,30 +18,27 @@ const ModalSizesMapper = {
   zen: '100vw',
```

Comment:
what a strange name. ☯️ Do we want to keep it?

Reply:
yes, it's used inside MarkdownEditor